algorithms/german/stemmer.tt

```diff
@@ -250,7 +250,7 @@ Step 2:
     </p>
         <DL><DD>
         (<I>a</I>) <B><I>en  &nbsp;  er  &nbsp;  est</I></B><BR>
-        (<I>b</I>) <B><I>st</I></B> (preceded by a valid <B><I>st</I></B>-ending, itself preceded by at least 3
+        (<I>b</I>) <B><I>st</I></B> (preceded by a valid <B><I>st</I></B>-ending, itself preceded by at least 3<BR>
```

Comment:
Hmm, this seems truncated and looking at the history it looks like recent edits lost " letters)" here.  I'll fix that too.

Your fixes look good.  (I already have a vague plan to move to not hand-writing HTML which is the root cause of these.)